src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -136,6 +137,15 @@ def go_from_trigger_mozilla_central(self):
         r.raise_for_status()
         push_id = r.json()['pushid']
 
+        for zf in self.artifactsHandler.get():
+            with zipfile.ZipFile(zf, 'r') as uz:
+                for i in uz.namelist():
+                    with uz.open(i, 'r') as fl:
+                        l = fl.read().decode('utf-8').splitlines()
```

Comment:
Maybe this would be more readable:
```Python
source_files = (line[3:] for line in fl if line.startswith('SF:'))
missing_files = [f for f in source_files if not os.path.exists(os.path.join(self.repo_dir, f))]
```